tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -1050,6 +1050,19 @@ def find_interface(tests, id=nil, skipcheck=true)
     msg = 'Unable to find suitable interface module for this test.'
     prereq_skip(tests[:resource_name], self, msg)
   end
+
+  if type =~ /ethernet/i
```

Comment:
The idea is that I want to work on an interface already selected by the first block and *ONLY* if that selected interface is on an N7K/F2 card (see the check on line 1058). if the other setup functions have caused a limit-resource to be with 'f3' cards, it would fail the guard below. So there should be no interaction with other setup methods.
The issue is specific to f2/f2e cards, so every L3 feature which chooses an f2/f2e interface would need to change. I thought this is a common function used by such features. So a common place to change.